### DISCUSSION TITLE
求助cyberdog刷机失败

### DISCUSSION THREAD

**Discussion by curui** [MiRoboticsLab/cyberdog_ros2#140] [General]
![Z1LUDCN$WC{Y5ABZ99V~NGB](https://user-images.githubusercontent.com/83493228/159952540-6cb6fbbc-067c-4105-bb33-6da835280618.png)  不知道什么原因一直报这个错误 环境是虚拟机!!!!!!!!!!

**Comment by curui** [comment]
经过测试  英伟达官方也做出解释 虚拟机环境不可以刷机    后来在单独装系统解决了问题

**Reply by yangyh75** [reply]
虚拟机可以线刷铁蛋，一直这样做.
